build.rs

```diff
@@ -83,12 +83,15 @@ fn main() {
     }
 
     let connection_testcases = testcases.join("connection_testcases.json");
+    if !connection_testcases.exists() {
+        return;
```

Comment:
Maybe we can give some warning here, so that we know the shared test cases are not running for a reason?

Reply:
Seems reasonable! I'll move these checks earlier and centralize the handling of this "missing submodules" case.

Reply:
@fantix [`d2ac70c` (#1100)](https://github.com/edgedb/edgedb-cli/pull/1100/commits/d2ac70ccf8bba56f67b15c43772492d7fae5523e) ⚡